ouroboros-network/src/Ouroboros/Network/Mux.hs

```diff
@@ -138,17 +145,25 @@ muxChannel pmss mid md w cnt =
         -- We send CBOR encoded messages by encoding them into by ByteString
         -- forwarding them to the 'mux' thread, see 'Desired servicing semantics'.
         --say $ printf "send mid %s mode %s" (show mid) (show md)
+        when (BL.length encoding > maximumMessageSize mid) $
```

Comment:
Improve comment here to note that this check is voluntary.

Reply:
Will address.